CHANGELOG.md

```diff
@@ -13,7 +13,8 @@ current changes on git with [previous release tags][git_tag_comparison].
 
 ### Added
 
-- [Bloom][6397]
+- [Added a new `run_criteria` example demonstrating run criterias.][7338]
```

Comment:
Can you revert the changes to `CHANGELOG.md`? This tends to be generated all in one batch at each release.

Reply:
indeed